webrtc.html

```diff
@@ -682,6 +682,14 @@
                     <p>If <var>connection</var>'s <a>signaling state</a>
                     is <code>closed</code>, then abort these steps.</p>
                   </li>
+                  
+                  <li>
+                    <p>If elements of the SDP returned by <code>createOffer</code>
+                    were modified before passing to <code>setLocalDescription</code>,
+                    and the modifications are invalid as specified in
```

Comment:
The algorithm is not specific to createOffer/setLD, so this should be reworded to apply more generically.